docs/_sources/vc_features.rst.txt

```diff
@@ -0,0 +1,36 @@
+VC features
+===========
+Conda builds Windows packages with features as a means of keeping the Visual Studio version (``vc``) used to build packages uniform across a given Python version.
+
+Rule of thumb
+-------------
+* if it's a shared library that something other than Python will use, you only need python in build to activate features. You need the features block;
```

Comment:
I think what he is saying is that py35 and py36 both use MSVC 2015. So for non-python things, there is no need to build against Python 3.5 and 3.6 as the both result in a package built with the same compiler, thus having the same feature, and the same name. We would save resources by skipping one. Please correct me if I have misunderstood @isuruf .